styles.css

```diff
@@ -493,3 +493,301 @@ textarea {
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .card-details {
+    width: 80%;
+    height: 50px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .project-btn {
+    margin-top: 0;
+    box-sizing: border-box;
+    padding: 16px;
+    background: #fff;
+    border: 2px solid #98a8cf;
+    border-radius: 16px;
+  }
+
+  .card2 {
+    background-image: url("images/desktop-Snapshoot-Portfolio2.svg");
+    background-repeat: no-repeat;
+    background-position: 660px 24px;
+    display: flex;
+  }
+
+  .desktop-card-details-left {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 100%;
+    height: 70%;
+    padding: 48px 0 80px 48px;
+  }
+
+  .card-details-left {
+    width: 45%;
+  }
+
+  .card3 {
+    background-image: url("images/desktop-Snapshoot-Portfolio3.svg");
+    background-repeat: no-repeat;
+    background-position: 24px 24px;
+  }
+
+  .card4 {
+    background-image: url("images/desktop-Snapshoot-Portfolio4.svg");
+    background-repeat: no-repeat;
+    background-position: 660px 24px;
+    display: flex;
+  }
+
+  .about {
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    width: 100%;
+  }
+
+  .about-part {
+    padding-left: 118px;
+  }
+
+  .about-title {
+    margin: 28px 0 0 0;
+  }
+
+  .about-details {
+    width: 450px;
+    height: 100px;
+  }
+
+  .languages-part {
+    padding-right: 142px;
+  }
+
+  .first-skill {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 24px 0 40px;
+  }
+
+  .second-skill {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 24px 0 40px;
+  }
+
+  .third-skill {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 24px 0 40px;
+  }
+
+  .all-languages {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .desktop-langages {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 12px;
+    padding-top: 12px;
+    height: 120px;
+    width: 122px;
+  }
+
+  .desktop-language-text {
+    margin: 12px 0 0 0;
+    padding-right: 20px;
+  }
+
+  .contact-form {
+    grid-column: 1/3;
+    background-image: url("images/contact\ form\ background\ shapes\(1\).svg");
+    width: 98.3%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 40px;
+  }
+
+  .contact-form-details {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .contact-title {
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin: 0;
+  }
+
+  .contact-details {
+    width: 40%;
+  }
+
+  .contact-form-list {
+    margin: 0;
+    width: 450px;
+  }
+
+  input {
+    width: 450px;
+    border-radius: 0;
+    padding-left: 15px;
+  }
+
+  textarea {
+    width: 450px;
+    border-radius: 0;
+    padding: 15px 0 0 15px;
+  }
+
+  .contact-btn {
+    align-self: center;
+  }
```

Comment:
- [ ] While you made a great effort to fix this issue with the contact form[ input](https://github.com/Hamdullah-Andar/HamiPortfolio/pull/4#discussion_r1089183916) and [button](https://github.com/Hamdullah-Andar/HamiPortfolio/pull/4/files#r1089194144) raised by the previous reviewer, I still see the same issue here. Kindly fix it

Your current design from `desktop-version` branch

![image](https://user-images.githubusercontent.com/84156714/215194980-067f6fc6-985d-48cf-8942-ddcebe9c20e1.png)